plugin/build.gradle.kts

```diff
@@ -124,11 +115,9 @@ license {
 
 publishing {
     publications {
-        register<MavenPublication>("plugin") {
+        create<MavenPublication>("pluginMaven") {
```

Comment:
@jbartok Is this the right way to customize the artifact ID and POM?

Reply:
Yes